statistics/testdata/trace_suite_out.json

```diff
@@ -0,0 +1,181 @@
+[
+  {
+    "Name": "TestTraceCE",
+    "Cases": [
+      {
+        "Expr": "a > 0 and a < 2",
+        "Trace": [
+          {
+            "TableID": 57,
+            "TableName": "",
+            "Type": "Table Stats-Expression-CNF",
+            "Expr": "`and`(`gt`(test.t.a, 0), `lt`(test.t.a, 2))",
+            "RowCount": 4
+          },
+          {
+            "TableID": 57,
+            "TableName": "",
+            "Type": "Table Stats-Expression-CNF",
+            "Expr": "`and`(`gt`(test.t.a, 0), `lt`(test.t.a, 2))",
+            "RowCount": 4
+          }
+        ]
+      },
+      {
+        "Expr": "a >= 1 and a < 10",
+        "Trace": [
+          {
+            "TableID": 57,
+            "TableName": "",
+            "Type": "Table Stats-Expression-CNF",
+            "Expr": "`and`(`ge`(test.t.a, 1), `lt`(test.t.a, 10))",
+            "RowCount": 6
+          },
+          {
+            "TableID": 57,
+            "TableName": "",
+            "Type": "Table Stats-Expression-CNF",
+            "Expr": "`and`(`ge`(test.t.a, 1), `lt`(test.t.a, 10))",
+            "RowCount": 6
+          }
+        ]
+      },
+      {
+        "Expr": "a < 3 or b < 4",
+        "Trace": [
+          {
+            "TableID": 57,
+            "TableName": "",
+            "Type": "Table Stats-Expression-CNF",
+            "Expr": "`lt`(test.t.a, 3)",
+            "RowCount": 6
+          },
+          {
+            "TableID": 57,
+            "TableName": "",
+            "Type": "Table Stats-Expression-CNF",
+            "Expr": "`lt`(test.t.a, 3)",
+            "RowCount": 6
```

Comment:
Should we remove duplicated records before exporting them?

Reply:
Yes. We will add deduplication logic in the next PRs.